src/components/donation/Steps/types.ts

```diff
@@ -53,9 +54,10 @@ export type DonationDetails =
 export type Mode = "live" | "preview";
 
 export type Init = {
+  source: DonationSource;
```

Comment:
since `config` can now be set by donate page (from db ) or the widget page (from widget params) - it is now not a good determinant of donation source. thus standalone  `source` attribute